core/templates/pages/exploration-editor-page/settings-tab/templates/preview-summary-tile-modal.component.ts

```diff
@@ -37,20 +37,20 @@ export class PreviewSummaryTileModalComponent extends ConfirmOrCancelModal {
   ) {
     super(ngbActiveModal);
   }
-  getExplorationTitle(): string {
+  getExplorationTitle(): string | boolean {
```

Comment:
Do we need to make all of these `string | boolean` like you've done here? Would it be possible to keep this typing as `string` since we know that exploration title is a string and not a boolean?

Reply:
Thanks, what you said is true for the `exploration-property` service in general, but specifically for the `preview-summary-tile-modal`, the exploration title is **always** a string. It makes sense and is much more clear if the return type were a string. Is that possible? @heyimShivam